src/Tizen.NUI/src/internal/DisposeQueue.cs

```diff
@@ -9,9 +9,11 @@
 
 using System;
 using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
 
 namespace Tizen.NUI
 {
+    [SuppressMessage("Microsoft.Design", "CA1001:Types that own disposable fields should be disposable")]
```

Comment:
How about adding `Justification` for the future developers?
For example,
```C#
[SuppressMessage("Microsoft.Design", "CA1001:Types that own disposable fields should be disposable", Justification="This is a singleton class and is not disposed")]
```

Reply:
Done